app/components/helpscout-beacon.js

```
@@ -0,0 +1,28 @@
+import Component from '@glimmer/component';
+import { loadScript } from '../utils/load-helpscout-beacon';
+import { action } from '@ember/object';
+
+export default class HelpScoutBeaconComponent extends Component {  
```

ESLint

[error] Delete `··`

(prettier/prettier)